l10n_it_account/__init__.py

```diff
@@ -0,0 +1,49 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import models
+from openupgradelib import openupgrade
+from psycopg2 import sql
+
+OLD_MODULE_NAME = "l10n_it_account_tax_kind"
+MODEL = "account.tax"
+OLD_MODEL = "account.tax.kind"
+RENAMED_FIELDS = [
+    (
+        "law_reference",
+        "l10n_it_law_reference",
+    ),
+]
+
+
+def _l10n_it_account_post_init(env):
+    if openupgrade.is_module_installed(env.cr, OLD_MODULE_NAME):
+        openupgrade.logged_query(
+            env.cr,
+            sql.SQL(f"""
+                UPDATE
+                    {MODEL.replace(".", "_")}
+                SET
+                    l10n_it_exempt_reason = kind_id.code
+                FROM
+                    {OLD_MODEL.replace(".", "_")} AS kind
+                WHERE
+                    {MODEL.replace(".", "_")}.kind_id = kind.id
+                    AND {MODEL.replace(".", "_")}.kind_id IS NOT NULL
+            """),
+        )
+
+        field_spec = []
+        for renamed_field in RENAMED_FIELDS:
+            old_field, new_field = renamed_field
+            field_spec.append(
+                (
+                    MODEL,
+                    MODEL.replace(".", "_"),
+                    old_field,
+                    new_field,
+                )
+            )
+        openupgrade.rename_fields(
+            env,
+            field_spec,
+        )
```

Comment:
Visto che `l10n_it_account` esiste anche nella 16, questo script sarebbe da mettere in `migrations`